Update AD-Forest-Recovery-Resetting-the-krbtgt-password.md (#4995)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Resetting-the-krbtgt-password.md

> [!NOTE]
> You should perform this operation twice. The password history of the krbtgt account is two, meaning it includes the two most recent passwords. By resetting the password twice you effectively clear any old passwords from the history, so there is no way another DC will replicate with this DC by using an old password.

> -> [!NOTE]
> + If you intent to reset the Key Distribution Center Service Account password twice. You must wait 10 hours between the password resets.
> +> [!NOTE]

Reviewer: ```suggestion

> [!NOTE]
> When resetting the Key Distribution Center Service Account password twice, a 10 hour waiting period is required between resets.
```
The formatting is correct, but I also suggest a more technical English phrasing to make it read more official than the previous suggestion. If you find my phrasing to be overly formal and technical, feel free to tell me so.
